docs/2_1_patching.md

```diff
@@ -8,16 +8,26 @@ The following pages will guide you through using ReVanced Manager to patch apps.
 2. Tap the + button in the bottom right corner
 3. Choose an app to patch[^1]
 4. Tap on the version of the app you want to patch[^2]
-5. Select the patches you want to apply
-6. Tap the Patch button
-7. Tap on the **Install** button
-   > **Note**: If you are rooted, you can mount the patched app on top of the original app.[^3]
-   > Optionally, you may export the patched app to storage using the options in the top right corner.
+5. Tap the 🪄 **Patch** button
+6. Tap on the 📲 **Install** button
+   > [!Note]
+   > If the **Install** Button isn't there, it could be that something went wrong. Refer to [3. ❔ Troubleshooting](3_troubleshooting.md)
+   > [!Tip]
```

Comment:
This does not seem to render correctly
![image](https://github.com/user-attachments/assets/b79b32d2-238a-4181-88c2-045fe56e75eb)

Reply:
Nested note blocks don't render unfortunately. See how the main branch solves this.